mlir-sycl/lib/Dialect/IR/SYCLOps.cpp

```diff
@@ -0,0 +1,115 @@
+//===--- SYCLOps.cpp ------------------------------------------------------===//
+//
+// MLIR-SYCL is under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Dialect/SYCL/IR/SYCLOps.h"
+
+#include "mlir/IR/OpImplementation.h"
+#include "llvm/ADT/TypeSwitch.h"
+
```

Comment:
Add `using namespace mlir::sycl;`?

Reply:
Thanks. I just didn't want to introduce many changes in this PR, as they'd be difficult to visualize (as the code is being moved around). I'll do this the following days.